Domain.Sql/SqlEventSourcedRepository{T}.cs

```diff
@@ -161,6 +163,11 @@ public class SqlEventSourcedRepository<TAggregate> : IEventSourcedRepository<TAg
         /// <exception cref="ConcurrencyException"></exception>
         public async Task Save(TAggregate aggregate)
         {
+            await Save(aggregate, new List<EventSourcedRepositoryMigrator.RenameRequest>());
+        }
+
+        async Task Save(TAggregate aggregate, IList<EventSourcedRepositoryMigrator.RenameRequest> pendingRenames)

```

Comment:
Does this need to be list? It looks like IEnumerable would work and is the least require interface for what the repository needs.